scripts/feature_def_gen/mirna_data/mirna_feature_def_builder.py

```diff
@@ -106,8 +106,6 @@ def build_table_mapping(self, config):
         return result
 
     def unpack_query_response(self, row_item_array):
-        table_config_mapping = self.build_table_mapping(self.config)
-
```

Comment:
This needs to be restored as well. If the variable is unused, it can be removed, but the call should remain.

Reply:
Thank you for the comments! I have gone through all the changes and fixed the mistakes. But for this one I am not sure if it should be added back. I looked into build_table_mapping() above and all it does is generate the result dictionary which is not used here. So I guess we do not need to call it?

Reply:
hm okay, let's toss this one then.